src/check_release.rs

```diff
@@ -356,6 +356,7 @@ pub(super) fn run_check_release(
         let suggested_bump = suggested_versions.iter().max().copied();
 
         if let Some(required_bump) = required_bump {
+            writeln!(config.stderr())?;
```

Comment:
Are you sure `config.shell_print()` goes to stderr, not stdout? We want the newline in the same stream.

Reply:
[It does.](https://github.com/obi1kenobi/cargo-semver-checks/blob/5108eeeffaf424e8d1a0019534414e7f11276a66/src/config.rs#L122).  Almost everything writes to stderr, except the `--- failure/warning ---` blocks. Does this newline belong to the `Summary` block or would it make more sense to write an extra newline after every `--- failure/warning ---` block?

Reply:
I'm open to it either way, whatever is cleaner to do.

Reply:
I think it makes more sense to put it with the summary block, as the `--- failure ---` blocks already separate themselves with newlines, so we would be adding two newlines in between each block.

Reply:
Sounds good to me!